examples/module-dependency-order.js

```diff
@@ -1,6 +1,11 @@
 // example - valid: true
 angular.module('myModule', ['ngAnimate', 'ngRoute', 'app', 'appFilters', 'ui.router']);
 
+// example - valid: true
+// with ES6 syntax:
+// import uiRouter from 'angular-ui-router';
```

Comment:
this is indeed a hack - this line does not pass eslint. @Gillespie59 how would you advise to deal with it?